charts/openstack-cluster/templates/control-plane/kubeadm-control-plane.yaml

```diff
@@ -107,6 +115,7 @@ spec:
           (include "openstack-cluster.kubeadmConfigSpec" (list . .Values.controlPlane.kubeadmConfigSpec) | fromYaml)
           (include "openstack-cluster.osDistroKubeadmConfigSpec" (list . ) | fromYaml)
           (include "openstack-cluster.patchConfigSpec"  (list .) | fromYaml)
+          (include "openstack-cluster.controlplane.kubeadmConfigSpec.containerdRestart" . | fromYaml)
```

Comment:
This is no longer required if `preKubeadmCommands` are added to `openstack-cluster.kubeadmConfigSpec.containerd`.